docs/tutorial/tutorial.md

```diff
@@ -191,7 +191,7 @@ Now we're passing down two props from Board to Square: `value` and `onClick`. Th
 </button>
 ```
 
-This means that when the square is clicked, it calls the onClick function that was passed by the parent. The `onClick` doesn't have any special meaning here, but it's popular to name handler props starting with `on` and their implementations with `handle`. Try clicking a square – you should get an error because we haven't defined `handleClick` yet. Add it to the Board class:
+This means that when the square is clicked, it calls the onClick function that was passed by the parent. The `onClick` in `renderSquare` doesn't have any special meaning here, but it's popular to name handler props starting with `on` and their implementations with `handle`. Try clicking a square – you should get an error because we haven't defined `handleClick` yet. Add it to the Board class:
```

Comment:
Can we just move this section about "no special meaning" above to where it's defined, and add a sentence here that *this* is where `onClick` has special meaning (because it's on an a built-in component like `<button>` rather than our custom `Square`).